appservice/src/intTest/groovy/nu/yona/server/CreateUserOnBuddyRequestTest.groovy

```diff
@@ -347,7 +347,9 @@ class CreateUserOnBuddyRequestTest extends AbstractAppServiceIntegrationTest
 				],
 				"goals":[
 					{
-						"activityCategoryName":"news"
+						"@class":"budgetGoal",
```

Comment:
Let's stick with this for now. We anyway need to handle goal changes differently, see [YD-165](http://jira.yona.nu/browse/YD-165).